### ISSUE TITLE
Add ALBUMARTIST tag in "simple" API

### ISSUE THREAD

**Issue by jcnoir** [taglib/taglib#649]
We use the TagLib's "simple" API (C bindings).
We make a GO wrapper for it : https://github.com/wtolson/go-taglib
This works fine. Thanks for the good work !
One thing we miss is the ALBUMARTIST tag in this simple API.
This tag is commonly used to identify albums when you want to set a specific artist in each track of the album.

Thanks again for this fine library !

**Comment by inizan-yannick** [comment]
I've added this tag in gobject wrapper : https://github.com/inizan-yannick/taglib-glib

**Comment by pdgendt** [comment]
Pull request #716 might help you

**Comment by inizan-yannick** [comment]
Thanks ☺
I've To update glib wrapper so

**Comment by pdgendt** [comment]
@inizan-yannick I've updated the pull request to also work with ogg FLAC

**Comment by scotchi** [comment]
Not sure what the prevailing thought on expanding the generic `Tag` interface is, but for binary compatibility reasons, this can't be considered for the master branch since it adds virtual functions to existing classes.  It could still be considered for the taglib2 branch, however.

**Comment by 0xcaff** [comment]
It looks like this is available for id3v2, asf, ape and mp4 through `PropertyMap`'s `ALBUMARTIST` key in master.

**Comment by supermihi** [comment]
Yes, even with taglib 1 you can simply use something like `file->properties()["ALBUMARTIST"]`

**Comment by simao** [comment]
Is there a way to access `PropertyMap` in `File` using the C bindings?

**Comment by ufleisch** [comment]
Not yet, and probably not in the upcoming 1.12 release. But I think that this would be useful and will consider it for a later release.

**Comment by simao** [comment]
Great! I'll wait then. thanks

**Comment by Enrique-porras** [comment]
https://github.com/taglib/taglib/issues/649#issue-102523185

**Comment by ufleisch** [comment]
Since version 2.0, it is possible to set properties via the C-bindings.

**Comment by simao** [comment]
Thank you!
